ld/emultempl/pe.em

```diff
@@ -126,9 +127,8 @@ fragment <<EOF
 #define DEFAULT_PSEUDO_RELOC_VERSION 1
 #endif
 
-#define DEFAULT_DLL_CHARACTERISTICS	(${cygwin_behavior} ? 0 : \
-					   IMAGE_DLL_CHARACTERISTICS_DYNAMIC_BASE \
-					 | IMAGE_DLL_CHARACTERISTICS_NX_COMPAT)
+#define DEFAULT_DLL_CHARACTERISTICS	(IMAGE_DLL_CHARACTERISTICS_DYNAMIC_BASE \
```

Comment:
Perhaps cygwin_behavior should be configured instead of this change.

Reply:
Could be. My intention was to enable DynamicBase and High Entropy ASLR on x86\_64 as well as we will need to make it working anyway and enabling it can help us reveal the actual issues ASAP, e.g. when we will be launching Cygwin tests.

Reply:
if it is not breaking aarch64-w64-mingw32 it is ok for now.

Reply:
For any other target than `*-pc-cygwin` the `cygwin_behavior` behavior is 0 thus DynamicBase and High Entropy ASLR is always enabled.